accountClient/accountClient.go

```diff
@@ -43,6 +43,18 @@ func (c *E3dbAccountClient) InternalGetClientAccount(ctx context.Context, client
 	return result, err
 }
 
+// InternalGetStripeID attempts to get the stripe id for the specified account id
+func (c *E3dbAccountClient) InternalGetStripeID(ctx context.Context, accountID string) (*InternalGetStripeIDResponse, error) {
```

Comment:
https://golang.org/doc/effective_go.html#Getters Not necessarily suggesting a change here as I also prefer `Get` in many places, but maybe worth noting the idiomatic go way of naming getters.

Reply:
@willmichael yes but in this case these are http clients so they're (very loosely) following the idiom of RESTful design